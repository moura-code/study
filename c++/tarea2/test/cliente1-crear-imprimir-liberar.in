crearCliente 19 Casimira Gonzalez 27
imprimirCliente
liberarCliente
crearCliente 31 Esmeralda Rodriguez 30
imprimirCliente
liberarCliente
Fin
