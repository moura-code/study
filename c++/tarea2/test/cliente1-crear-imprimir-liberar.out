Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> Cliente creado de forma exitosa.
2> Cliente Casimira Gonzalez
Id: 19
Edad: 27
3> Cliente liberado con exito.
4> Cliente creado de forma exitosa.
5> Cliente Esmeralda Rodriguez
Id: 31
Edad: 30
6> Cliente liberado con exito.
7> Fin.
